examples/web-ui/material/menus/menuDesktop.smash

```diff
@@ -0,0 +1,9 @@
+* Set Desktop
+    * Click first level page button {{page}}
+        Log "Desktop click"
+            Click '{{page}}Button'
+
+    * Click second level page button {{page}}
+        Log "Desktop click"
+            Wait '2' secs
```

Comment:
`Wait` shouldn't be necessary (and should be avoided whenever possible). Does it not work without it? `Click` should be waiting for up to 2 secs for that element to appear. Otherwise, you may want to put in a `Wait until` step (more reliable than just `Wait`).